### ISSUE TITLE
Crash in `Tween.follow_property`

### ISSUE THREAD

**Issue by qarmin** [godotengine/godot#52831]
### Godot version

3.4.beta.custom_build. 316b5dad2

### System information

Ubuntu 21.04 - Nvidia GTX 970, Gnome shell 3.38 X11

### Issue description

When executing
```
	var temp_variable = Tween.new()
	temp_variable.get_runtime()
	temp_variable.remove_all()
	temp_variable.follow_property(null, NodePath("."), null, RichTextLabel.new(), NodePath("."), -33.532339, 47, 4, 28.430223)
```

Godot crashes with backtrace
```
[1] /lib/x86_64-linux-gnu/libc.so.6(+0x41040) [0x7f25ceb42040] (??:0)
[2] Object::get(StringName const&, bool*) const (/mnt/Miecz/godot3.2/core/object.cpp:488)
[3] Object::get_indexed(Vector<StringName> const&, bool*) const (/mnt/Miecz/godot3.2/core/object.cpp:626)
[4] Tween::follow_property(Object*, NodePath, Variant, Object*, NodePath, float, Tween::TransitionType, Tween::EaseType, float) (/mnt/Miecz/godot3.2/scene/animation/tween.cpp:1567 (discriminator 3))
[5] MethodBind9R<bool, Object*, NodePath, Variant, Object*, NodePath, float, Tween::TransitionType, Tween::EaseType, float>::call(Object*, Variant const**, int, Variant::CallError&) (/mnt/Miecz/godot3.2/./core/method_bind_ext.gen.inc:3153 (discriminator 40))
[6] Object::call(StringName const&, Variant const**, int, Variant::CallError&) (/mnt/Miecz/godot3.2/core/object.cpp:918 (discriminator 1))
[7] Object::callv(StringName const&, Array const&) (/mnt/Miecz/godot3.2/core/object.cpp:828)
[8] MethodBind2R<Variant, StringName const&, Array const&>::call(Object*, Variant const**, int, Variant::CallError&) (/mnt/Miecz/godot3.2/./core/method_bind.gen.inc:1717 (discriminator 12))
[9] Object::call(StringName const&, Variant const**, int, Variant::CallError&) (/mnt/Miecz/godot3.2/core/object.cpp:918 (discriminator 1))
[10] Variant::call_ptr(StringName const&, Variant const**, int, Variant*, Variant::CallError&) (/mnt/Miecz/godot3.2/core/variant_call.cpp:1175 (discriminator 1))
[11] GDScriptFunction::call(GDScriptInstance*, Variant const**, int, Variant::CallError&, GDScriptFunction::CallState*) (/mnt/Miecz/godot3.2/modules/gdscript/gdscript_function.cpp:1044)
[12] GDScriptInstance::call(StringName const&, Variant const**, int, Variant::CallError&) (/mnt/Miecz/godot3.2/modules/gdscript/gdscript.cpp:1169)
[13] Object::call(StringName const&, Variant const**, int, Variant::CallError&) (/mnt/Miecz/godot3.2/core/object.cpp:899 (discriminator 1))
[14] Variant::call_ptr(StringName const&, Variant const**, int, Variant*, Variant::CallError&) (/mnt/Miecz/godot3.2/core/variant_call.cpp:1175 (discriminator 1))
[15] GDScriptFunction::call(GDScriptInstance*, Variant const**, int, Variant::CallError&, GDScriptFunction::CallState*) (/mnt/Miecz/godot3.2/modules/gdscript/gdscript_function.cpp:1049)
[16] GDScriptInstance::call_multilevel(StringName const&, Variant const**, int) (/mnt/Miecz/godot3.2/modules/gdscript/gdscript.cpp:1184)
[17] Node::_notification(int) (/mnt/Miecz/godot3.2/scene/main/node.cpp:54)
[18] Node::_notificationv(int, bool) (/mnt/Miecz/godot3.2/./scene/main/node.h:45 (discriminator 14))
[19] Object::notification(int, bool) (/mnt/Miecz/godot3.2/core/object.cpp:929)
[20] SceneTree::_notify_group_pause(StringName const&, int) (/mnt/Miecz/godot3.2/scene/main/scene_tree.cpp:973)
[21] SceneTree::idle(float) (/mnt/Miecz/godot3.2/scene/main/scene_tree.cpp:528 (discriminator 3))
[22] Main::iteration() (/mnt/Miecz/godot3.2/main/main.cpp:2181)
[23] OS_X11::run() (/mnt/Miecz/godot3.2/platform/x11/os_x11.cpp:3641)
[24] /usr/bin/godot(main+0x125) [0x179733b] (/mnt/Miecz/godot3.2/platform/x11/godot_x11.cpp:57)
[25] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xd5) [0x7f25ceb29565] (??:0)
[26] /usr/bin/godot(_start+0x2e) [0x179715e] (??:?)

```

This example was found by fuzzer, so I don't recommend trying understand in what situation such code could be used in real project, because such situation probably doesn't exists

### Steps to reproduce

Above

**Comment by KoBeWi** [comment]
`follow_property` doesn't exist in 4.0, so this is only relevant for the 3.x branch.

**Comment by akien-mga** [comment]
Fixed by #52925.
